### PR TITLE
Added tips to FAQ and "how to" guide

### PR DESCRIPTION
**Pull request by mikej888**
Added to FAQ - 'Why I should allow my researcher to attend a boot camp?'

Added to "how to" guide...
- 'Why allow externals to come to your boot camp'
- 'Why pay instructors expenses'
- 'Ways to reduce the costs of your boot camp' 
  ...with links from the main body of the text

### CODE REVIEW

File: about/faq.html [hunk 1]
```diff
@@ -133,6 +134,14 @@
 our <a href="{{root_path}}/blog/">blog</a> and
 on <a href="{{twitter_url}}">Twitter</a>.</p>
 
+<p id="why-allow-attend-boot-camp"><em><strong>Why should I allow my researcher to attend a boot camp?</strong></em></p>
```
Comment (gvwilson) [inline about/faq.html]: "my researcher" sounds odd to North American ears --- do you mean, if I'm a prof or manager, why should I let my grad students or scientists attend boot camps?

File: bootcamps/how-to.html [hunk 2]
```diff
@@ -112,6 +112,10 @@
   </p>
 
   <p>
+    You may, however, wish to consider our reasons as to  why you should <a href="#allow-externals">allow externals to come to your boot camp</a>, even if they attend as observers.
```
Comment (gvwilson) [inline bootcamps/how-to.html]: "our reasons as to why" seems long-winded

File: bootcamps/how-to.html [hunk 3]
```diff
@@ -157,7 +161,8 @@
     and do not need to be paid for their time,
     but boot camp hosts are expected to cover their travel and accommodation costs.
     (As far as possible,
-    we try to match instructors to nearby sites in order to reduce these.)
+    we try to match instructors to nearby sites in order to reduce these,
+    but please see our reasons <a href="#pay-instructor-costs">why you should pay instructors' expenses</a>).
```
Comment (gvwilson) [inline bootcamps/how-to.html]: please see _why you should pay instructors' expenses_ (no need for "our reason")

File: bootcamps/how-to.html [hunk 4]
```diff
@@ -176,6 +181,10 @@
   </ul>
 
   <p>
+    You may also want to consider our suggestions as to <a href="#ways-to-reduce-costs"> ways to reduce the costs of running your boot camp</a>.
```
Comment (gvwilson) [inline bootcamps/how-to.html]: replace "our suggestions as to" with "these", and remove the space between the opening of the <a> and the word "ways"

**Comment by wking** [comment]
On Fri, Mar 29, 2013 at 10:15:44AM -0700, mikej888 wrote:

> - Why pay instructors expenses

Should be “pay to deliver a service” not “pay to delivery a service”.

**Comment by wking** [comment]
On Fri, Mar 29, 2013 at 10:15:44AM -0700, mikej888 wrote:

> - Ways to reduce the costs of your boot camp

I think you should use `&hellip;` or a UTF-8 `…` instead of `...`.
The current build system doesn't like non-ASCII codes, so that means
`&hellip;` for now.

**Pull request merged**